charts/firecloudui/templates/deployment.yaml

```diff
@@ -0,0 +1,99 @@
+{{- $imageTag := .Values.app.image.tag | default .Values.global.applicationVersion -}}
+{{- $legacyResourcePrefix := .Values.legacyResourcePrefix | default .Values.name -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}-deployment
+  labels:
+{{ include "firecloudui.labels" . | indent 4 }}
+spec:
+  revisionHistoryLimit: 0
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      deployment: {{ .Values.name }}-deployment
+  template:
+    metadata:
+      labels:
+        deployment: {{ .Values.name }}-deployment
+{{ include "firecloudui.labels" . | indent 8 }}
+      annotations:
+        {{- /* Automatically restart deployments on config map change: */}}
+        checksum/{{ .Values.name }}-configmap: {{ include (print $.Template.BasePath "/config/configmap.yaml") . | sha256sum }}
+    spec:
+      serviceAccountName: {{ .Values.name }}-sa
+      # Containers are configured to talk to each other by name
+      # via docker-compose links; make corresponding aliases
+      # to loopback in /etc/hosts
+      hostAliases:
+      - ip: 127.0.0.1
+        hostnames:
+        - app
+      volumes:
+      - name: app-ctmpls
+        secret:
+          secretName: {{ $legacyResourcePrefix }}-app-ctmpls
+      containers:
+      - name: {{ .Values.name }}-app
+        image: "{{ .Values.app.image.repository }}:{{ .Values.app.image.tag | default .Values.global.applicationVersion }}"
+        ports:
+          - containerPort: 443
+          - containerPort: 80
+          - containerPort: 8888
+        resources: # Mimic existing GCE vm requirements
```

Comment:
We've standardized on defining the resources stanza using [this syntax](https://github.com/broadinstitute/terra-helm/blob/47ecd9c540b4b07ce8844d3f57c1ef89c1ebb5e9/charts/firecloudorch/templates/deployment.yaml#L103) and then setting defaults in [values.yaml](https://github.com/broadinstitute/terra-helm/blob/47ecd9c540b4b07ce8844d3f57c1ef89c1ebb5e9/charts/firecloudorch/values.yaml#L45)

Reply:
Nice. Can make this change.

Amusingly, [this](https://github.com/broadinstitute/terra-helm/blob/47ecd9c540b4b07ce8844d3f57c1ef89c1ebb5e9/charts/firecloudorch/templates/deployment.yaml#L160) is right below your syntax link! 😂